frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/os-flavor.tsx

```diff
@@ -46,9 +46,11 @@ export const OSFlavor: React.FC<OSFlavorProps> = React.memo(
       concatImmutableLists(iGetLoadedData(commonTemplates), iGetLoadedData(userTemplates)),
     );
 
-    const operatingSystems = openshiftFlag
-      ? ignoreCaseSort(getOperatingSystems(vanillaTemplates, params), ['name'])
-      : operatingSystemsNative;
+    const operatingSystems = removeOSDups(
+      openshiftFlag
+        ? ignoreCaseSort(getOperatingSystems(vanillaTemplates, params), ['name'])
```

Comment:
is this the only place we should worry about?

Reply:
The template validations are working
except the wizard, do you think on other places ?

Reply:
IMO this functionality should  be implemented in the following function to be applied everywhere in the wizard (for example this is used in template prefil)

`getTemplateOperatingSystems = (templates: TemplateKind[])` in  `selectors/vm-template/advanced.ts`